File: src/main/scala/im/tox/toktok/app/Group.scala
package im.tox.toktok.app

import android.graphics.Color

import scala.collection.mutable.ListBuffer

class Group(name: String, list: ListBuffer[Friend], color: Int, statusColor: Int) {

  def getGroupName(): String = {
    return name
  }

  def getFriendsList(): ListBuffer[Friend] = {
    return list
  }

  def getPrimaryColor(): Int = {
    return color
  }

  def getStatusColor(): Int = {
    return statusColor
  }

}

object Group {
  val group = new Group(" \uD83C\uDF20 The Amazing Group", ListBuffer(Friend.lorem, Friend.john), Color.parseColor("#9B9B9B"), Color.parseColor("#5A5A5A"))
}

File: sources/app/src/main/java/im/tox/toktok/app/domain/Group.java
package im.tox.toktok.app.domain;

import android.graphics.Color;

import java.util.Arrays;
import java.util.List;

public final class Group {
    public static final Group group = new Group(
            " \uD83C\uDF20 The Amazing Group",
            Arrays.asList(
                    Friend.lorem,
                    Friend.john
            ),
            Color.parseColor("#9B9B9B"),
            Color.parseColor("#5A5A5A")
    );

    public final String groupName;
    public final List<Friend> friendsList;
    public final int primaryColor;
    public final int statusColor;

    public Group(
            String groupName,
            List<Friend> friendsList,
            int primaryColor,
            int statusColor
    ) {
        this.groupName = groupName;
        this.friendsList = friendsList;
        this.primaryColor = primaryColor;
        this.statusColor = statusColor;
    }
}
